src/extension.ts

```diff
@@ -38,8 +39,11 @@ export function activate(context: vscode.ExtensionContext) {
 				console.log("File has been created");
 			});
 
-		}).catch((err:any) => {
+		}).catch((err) => {
 			console.log("ERROR: " + err);
+			if(err === ECONNREFUSED) {
```

Comment:
`err instanceof ECONNREFUSED` seems to return an error that the 
```
The right-hand side of an 'instanceof' expression must be of type 'any' or of a type assignable to the 'Function' interface type.ts(2359)
```
Hence the triple ===

Reply:
ECONNREFUSED can also mean firewall

Reply:
Should we just warn the user for the set of these
1) The URL Entered could be incorrect
2) The Token Could be invalid
3) Check Firewall and retry

Reply:
if token is invalid Jenkins should not throw `ECONNREFUSED` rather a `401` as unauthorized.
That seems like a bug in Jenkins.

Reply:
No Jenkins handles it correctly: `Error 401 Invalid password/token for user: joseph petersen`

Reply:
So I guess for the time being the user should know what could be the possible reasons.The above three marginally cover the scenarios. What do you think ?

Reply:
No ECONNREFUSED only covers URL being inccorect or Firewall rules.

Reply:
Okay cool, will incorporate them